### PR TITLE
Nightlies for garden dependents of ign-rendering

### PR DESCRIPTION
**Pull request by scpeters**
Part of https://github.com/ignition-tooling/release-tools/issues/545

### CODE REVIEW

File: plugins/config/repository.yaml [hunk 1]
```diff
@@ -24,6 +24,43 @@ repositories:
...
+          - name: osrf
+            type: prerelease
+          - name: osrf
+            type: nightly
+    - name: ignition-gui7
+      repositories:
+          - name: osrf
+            type: stable
+          - name: osrf
+            type: prerelease
```
Comment (chapulina) [inline plugins/config/repository.yaml:49]:
```suggestion
            type: nightly
```

File: plugins/config/repository.yaml [hunk 2]
```diff
@@ -24,6 +24,43 @@ repositories:
...
+          - name: osrf
+            type: nightly
+    - name: ignition-gazebo7
+      repositories:
+          - name: osrf
+            type: stable
+          - name: osrf
+            type: prerelease
+          - name: osrf
+            type: nightly
```
Comment (chapulina) [inline plugins/config/repository.yaml:43]: Mixing prerelease and nightly goes :boom: Let's keep just nightly

**Review by chapulina — CHANGES_REQUESTED** [review]

File: plugins/config/repository.yaml (hunk 2 above)
Comment (scpeters) [reply plugins/config/repository.yaml:39]: e997bd10b8fe6ba13a024378af045adf44c6eaed

File: plugins/config/repository.yaml (hunk 1 above)
Comment (scpeters) [reply plugins/config/repository.yaml]: e997bd10b8fe6ba13a024378af045adf44c6eaed

**Review by chapulina — APPROVED** [review]

**Pull request merged**